README.md

```diff
@@ -60,7 +60,7 @@ instance as `received_logger`.
 ```ruby
 require 'optional_logger'
 
-logger = OptionalLogger.new(received_logger)
+logger = OptionalLogger::Logger.new(received_logger)
```

Comment:
I can't buy into either of those as they both seem like entities so it sounds like it is a logger of options not an optional logger.

I am not a fan of the `Logger::Logger` which is what I am guessing you are trying to get away from. So, I am open to other suggestions.